changelog

```diff
@@ -1,3 +1,240 @@
+      * The Ka'het Nullifier has been rebalanced to more effective disable targets by draining their energy. (@beccabunny)
+        * Energy damage from 0 to 3200. This is energy which is immediately removed from the target's batteries on impact.
+        * Ion damage from 150 to 220. This is lingering energy loss that results in about 22,000 energy lost over the course of the next ten seconds after impact.
+        * Scrambling damage from 150 to 10. This causes the Nullifier to have much less of an effect on the target's weapons, instead relying on its much greater energy and ion damage.
+      * Buffed the Ion Cannon's ion and scrambling damage by 20%, from 5 to 6 for each in order to make it a bit more competitive with other weaponry. (@Hurleveur)
+      * Reduced the profitability of commodity trading in Hai space by about 40% by bringing trade prices closer together, as the lack of piracy in Hai space makes it an easy place for trading. (@Anarchist2)
+      * Combined the two Barb variants into a single ship. This effectively just means that the turreted variant of the ship has gained +4 weapon space, and the Proton Gun variant of the ship is no longer sold. These two ships weren't sufficiently differentiated from one another to justify keeping both around. (@Quantumshark)
+      * Reduced the buff to engine space provided to the Marauder Leviathan and Falcon engine variants. (@Quantumshark)
+        * Falcon: +55 to +25 (total of 220 to 190)
+        * Leviathan: +80 to +40 (total of 220 to 180)
+    * Graphics:
+    * Other:
+      * Tweaked the wording of the description of the Vivid Aether planet. (@Zitchas)
+      * Ringworld planets now have a "ringworld" attribute for missions to use. (@Quantumshark)
+      * Added a "frozen" attribute to various cold and frozen worlds for missions and spaceport news to use. (@dorbarker, @Anarchist2)
+      * The Unfettered Solifuge and Violin Spider are now added to the Unfettered shipyards at the same time as they are added to Unfettered fleets. (@Hurleveur)
+      * Added the "forest" attribute to Stormhold and changed its planet sprite in order to better match the planet's description. (@bene-dictator)
+      * Various NPC ships in Free Worlds missions which state that they carry cargo now actually have the cargo you're told they have. (@bene-dictator)
+      * Changed the attitude of various Hai governments toward certain actions that the player takes to better reflect the story. (@Hurleveur)
+        * You no longer gain reputation with the Hai for attacking or destroying Unfettered ships.
+        * You no longer lose reputation with the Unfettered for disabling Unfettered ships.
+        * You now gain 50% more reputation for assisting Hai ships.
+      * Standardized the language used for attribute tooltips.
+      * Renamed the Hai's Railgun to the Skipper Railgun. (@Saugia)
```

Comment:
```suggestion
      * Renamed the Hai Railgun to the Skipper Railgun using the "display name" attribute. (@Saugia)
```

Reply:
The changelog doesn't need to explain how something was done.

Reply:
OK:
```suggestion
      * Renamed the Hai Railgun to the Skipper Railgun. (@Saugia)
```
I do think "Hai" is better than "Hai's", though, as it is the actual outfit name.

Reply:
> I do think "Hai" is better than "Hai's", though, as it is the actual outfit name.

Except it's not. The gun itself is just the "Railgun" with the display name of "Skipper Railgun". "Hai" isn't a part of the name.